src/components/MovieCard/Additions.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import style from './Additions.module.scss'
+import styles from './MovieCard.module.scss'
+import Button from '../Button/Button'
+import Portal from '../Portal/Portal'
+import EditMovie from '../EdditMovie/EditMovie'
+import DeletionConfirmation from '../DeletionConfirmation/DeletionConfirmation'
+
+function Additions() {
+  const [visible, setVisible] = React.useState(false)
+  const [showModal, setShowModal] = useState(false)
+  const [showModalDelete, setShowModalDelete] = useState(false)
+  const handleClickClose = () => {
+    setVisible(!visible);
+  };
+
+  return (
+      <div className={style.additions_wrapper}>
+        <div className={`${style.additions_menu} ${styles.additions_menu}`}
+             onClick={() => setVisible(!visible)}>
+          <p className={style.point}></p>
+          <p className={style.point}></p>
+          <p className={style.point}></p>
+        </div>
+        <div className={style.additions_choose} style={{display: visible ? 'block' : 'none'}}>
```

Comment:
it's better to handle this with `conditional rendering` https://reactjs.org/docs/conditional-rendering.html